information-system/provenance/overview/what-are-entities.md

```diff
@@ -8,14 +8,58 @@ parent: Provenance Overview
 ---
 # What are entities?
 
-In the RRAP M&DS Information System (IS), entities are things that are registered in the M&DS IS Registry. Registering entities is fundamental to establishing stable provenance. 
+## 1. Overview
 
-Each registered item (entity) is given a unique and persistent identifier (ID). These IDs provide a mechanism for 2 things:
+In the RRAP M&DS Information System (IS), entities are things that are registered in the M&DS IS Registry. Registration of entities is fundamental to establishing stable provenance. It establishes a record for each entity used or generated in producing or delivering an artifact in a provenance record. 
```

Comment:
'registered' --> 'recorded' ?

A key outcome of 'registration' is the allocation of an identifier in the context of the registry. This is essential for cross-referencing and persistence, which is a pre-requisite for formalizing a provenance trace.

Reply:
Are model runs also registered? Need to be careful about use of term 'Entities'. `prov:Entity` is a continuant. do you intend that registered entities include both Continuants and Occurrents?

Reply:
yes, we're mixing the definition of prov:Entity and entities in the everyday general sense... i had raised this issue with @PeterBaker0 

agree the semantics are different if we're including registration of Occurrents (i.e. Activity > Model Run). 

however, at the time, it was felt that entities in the everyday general sense was sufficient. maybe not. what would be an alternative? I suggested "Thing" but that might be too general... Is there a suitable alternative?

Reply:
We could substitute like this:
- Entity -> Registered item
- Entity registry -> Registry

Reply:
I just helped Peter with some registry description words. 
We used 'Thing'.

Reply:
(Interesting that we are re-discovering the ISO 19135 model that was implemented in LDR ...)